### ISSUE TITLE
More interactions if more cells - built in correction?

### ISSUE THREAD

**Issue by michaelmather15** [ventolab/CellphoneDB#203]
Hello,
Thanks for developing this excellent tool. I have noticed that when analysing larger populations of cells I see more significant interactions. I'm not sure whether this is simply bound to be the case statistically where more cells -> more likely to have interactions. Or whether this is true variation between biological groups i.e. a particular cell type in group A has a different number of interactions to the same cell type in group B (regardless of the number of cells in each).

In other words, is there in-built statistical correction for number of cells analysed?

Thanks,
Michael

**Comment by ktroule** [comment]
Hi.

If you are concerned about potential biases due to population size, you can downsample the largest to the same number of cells of the other before proceeding with the analysis. However, as you point, this could be due to differences between groups.

Regards
